apps/sms/views/shared/js/utils.js

```diff
@@ -473,6 +473,25 @@
       });
       return parsed;
     },
+    url(base, params) {
+      if (base.indexOf('?') === -1) {
```

Comment:
Wondering if we can use [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) and [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) here?